extension/apmconfigextension/internal/examples/go.mod

```diff
@@ -0,0 +1,30 @@
+module github.com/elastic/opentelemetry-collector-components/extension/apmconfigextension/internal/examples
+
+go 1.23.6
+
+require (
+	github.com/google/uuid v1.6.0
+	github.com/knadh/koanf v1.3.3
+	github.com/open-telemetry/opamp-go v0.19.0
+
+)
+
+require (
+	github.com/cenkalti/backoff/v4 v4.3.0 // indirect
+	github.com/davecgh/go-spew v1.1.1 // indirect
+	github.com/gorilla/websocket v1.5.3 // indirect
+	github.com/kr/pretty v0.3.1 // indirect
+	github.com/mitchellh/copystructure v1.2.0 // indirect
+	github.com/mitchellh/mapstructure v1.4.1 // indirect
+	github.com/mitchellh/reflectwalk v1.0.2 // indirect
+	github.com/pmezard/go-difflib v1.0.0 // indirect
+	github.com/rogpeppe/go-internal v1.11.0 // indirect
+	github.com/stretchr/testify v1.10.0 // indirect
+	golang.org/x/sys v0.19.0 // indirect
+	google.golang.org/protobuf v1.36.2 // indirect
+	gopkg.in/check.v1 v1.0.0-20201130134442-10cb98267c6c // indirect
+	gopkg.in/yaml.v2 v2.2.8 // indirect
+	gopkg.in/yaml.v3 v3.0.1 // indirect
+)
+
+replace github.com/elastic/opentelemetry-collector-components/extension/apmconfigextension => ../../
```

Comment:
Is there a reason for this absolute pathed replace statement?

Reply:
example removed